src/core/src/Eryph.VmConfig.Primitives/Resources/Machines/CatletStopMode.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Eryph.Resources.Machines;
+
+[JsonConverter(typeof(JsonStringEnumConverter))]
+public enum CatletStopMode
+{
+    /// <summary>
+    /// This mode attempts to gracefully shut down the catlet.
+    /// </summary>
+    Shutdown = 0,
+
+    /// <summary>
+    /// This mode immediately stops the catlet comparable to pulling the power plug.
+    /// </summary>
+    Hard = 1,
+
+    /// <summary>
```

Comment:
please comment this out, so it is not added to API

Reply:
Fixed